ch08.adoc

```diff
@@ -675,3 +675,130 @@ The data creator shall specify the floating-point arithmetic precision used duri
+Thus **`quantization_nsb`** is an integer type attribute with **`1 \<= NSB \<= 23`** for data type **`float`** or **`real`**, and **`1 \<= NSB \<= 52`** for data type **`double`**.
+
+The BitGroom, Granular BitGroom, and DigitRound algorithms guarantee preservation of a specified number of significant digits (NSD) in base 10 representation.
+The actual number of mantissa bits quantized depends on the algorithm.
+Thus all data variables quantized by BitGroom, Granular BitGroom, or DigitRound must have a corresponding attribute **`quantization_nsd`**.
+The value of **`quantization_nsd`** is an integer with **`1 \<= NSD \<= 7`** for data type **`float`** or **`real`**, and **`1 \<= NSD \<= 15`** for data type **`double`**.
+
+[[example-quantization-nsb-libnetcdf]]
+[caption="Example 8.8. "]
+.Quantization performed by BitRound algorithm in libnetcdf
+====
+----
+  variables:
+    char quantization_info ;
+      quantization_info:algorithm = "bitround" ;
+      quantization_info:implementation = "libnetcdf version 4.9.2" ;
+
+    float ps(time,lat,lon) ;
+      ps:_QuantizeBitRoundNumberOfSignificantBits = 9 ;
+      ps:quantization = "quantization_info" ;
+      ps:quantization_nsb = 9 ;
+      ps:standard_name = "surface_air_pressure" ;
+      ps:units = "Pa" ;
+----
+Note how the same NSB is reported in two attributes of the data variable **`ps`**. The quantization variable (**`quantization_info`**) **`implementation`** attribute reveals that the netCDF library applied the BitRound algorithm. The netCDF library wrote the system-defined **`_QuantizeBitRoundNumberOfSignificantBits`** attribute [<<NetCDF>>] which contains the same parameter value as the CF **`quantization_nsb`** attribute.
+====
+
+[[example-quantization-nsd-multiple-variables-nco]]
+[caption="Example 8.9. "]
+.Quantization performed by Granular BitGroom algorithm in NCO
+====
+Quantization of different variables to different levels often makes good scientific sense. Here the pressure variable **`ps`** has four significant digits of precision while the temperature variable **`ts`** retains only three significant digits. 
+----
+  variables:
+    char quantization_info ;
+      quantization_info:algorithm = "granular_bitround" ;
+      quantization_info:implementation = "NCO version 5.2.7" ;
+
+    float ps(time,lat,lon) ;
+      ps:standard_name = "surface_air_pressure" ;
+      ps:units = "Pa" ;
+      ps:quantization = "quantization_info" ;
+      ps:quantization_nsd = 4 ;
+
+    float ts(time) ;
+      ts:standard_name = "surface_temperature" ;
+      ts:units = "K" ;
+      ts:quantization = "quantization_info" ;
+      ts:quantization_nsd = 3 ;
+----
+Both variables were quantized by the same algorithm and so utilize the same quantization variable. **`quantization_info`** reveals that the Granular BitRound algorithm in NCO performed the quantization. Since the netCDF library did not perform the quantization, there is no system-defined long quantization attribute.
```

Comment:
```suggestion
Both variables were quantized by the same algorithm and so utilize the same quantization variable.
**`quantization_info`** reveals that the Granular BitRound algorithm in NCO performed the quantization.
Since the netCDF library did not perform the quantization, there is no system-defined underscored quantization attribute.
```